src/components/login/container/index.jsx

```diff
@@ -40,12 +30,24 @@ export class Login extends Component {
     }
   }
 
+  handleInput = (event) => {
+    const { name, value } = event.target;
+    this.setState({ [name]: value });
+  };
```

Comment:
Do all these changes have anything to do with the ticket?

Reply:
Thank you @dkayiwa for this. Let me revert all these unrelated changes.